package.json

```diff
@@ -9,13 +9,13 @@
   },
   "scripts": {
     "unit": "jasmine test/test.js",
-    "debug-unit": "DEBUG_TEST=true node --inspect-brk ./node_modules/.bin/jasmine test/test.js",
```

Comment:
out of curiosity, why changing `.bin/jasmine` to `jasmine/bin/jasmine.js`?

Reply:
I had an exception:
```
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^
SyntaxError: missing ) after argument list
```

This is where I found the solution https://github.com/gotwarlost/istanbul#usage-on-windows